<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>873</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="stack_send_file">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>:/img/resources/logo.png</pixmap>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stack_analyze_file">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="all_analysis_tabs">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_nt_header">
           <attribute name="title">
            <string>Header</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="QTableWidget" name="table_nt_header_widget">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::DoubleClicked</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QTextEdit" name="textEdit_2">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;stack analyze&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_imports">
           <attribute name="title">
            <string>Imports</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_nt_imports"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_sections">
           <attribute name="title">
            <string>Sections</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_sections">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_strings_classic">
           <attribute name="title">
            <string>Mega Strings Miner</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>General</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinbox_size_strings_classic">
                 <property name="toolTip">
                  <string>If list is big enter text instead of using spin button</string>
                 </property>
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QCheckBox" name="checkbox_utf8">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>utf8</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="8">
                <widget class="QCheckBox" name="chk_regex_string_classic">
                 <property name="text">
                  <string>regex</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QLineEdit" name="txt_filter"/>
               </item>
               <item row="0" column="6">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Filter</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="9">
                <widget class="QPushButton" name="btn_search_string_classic">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QCheckBox" name="checkbox_case_sensitive">
                 <property name="text">
                  <string>case sensitive</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QCheckBox" name="checkbox_utf16">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>utf16</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Min. size</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="10">
                <widget class="QPushButton" name="btn_switch_string_classic">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Edition</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="grp_string_hunting">
              <property name="title">
               <string>Strings Hunter</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_20">
               <item row="0" column="2">
                <widget class="QRadioButton" name="chk_regex_ip">
                 <property name="text">
                  <string>IPV4</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="chk_disable_string_hunt">
                 <property name="text">
                  <string>Disable</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QRadioButton" name="chk_regex_unix_path">
                 <property name="text">
                  <string>Unix path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QRadioButton" name="chk_regex_url">
                 <property name="text">
                  <string>URL</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QRadioButton" name="chk_regex_win_path">
                 <property name="text">
                  <string>Windows path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QRadioButton" name="chk_regex_registry">
                 <property name="text">
                  <string>Registry</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QRadioButton" name="chk_regex_email">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QRadioButton" name="chk_regex_base64">
                 <property name="text">
                  <string>Base64</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QStackedWidget" name="stack_strings_classic">
              <widget class="QWidget" name="page">
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="2" column="0">
                 <widget class="QListWidget" name="list_strings_classic">
                  <property name="autoScroll">
                   <bool>true</bool>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::AllEditTriggers</set>
                  </property>
                  <property name="viewMode">
                   <enum>QListView::ListMode</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="sortingEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="0">
                 <widget class="QPlainTextEdit" name="text_strings_classic"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>Noise remover</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_19">
               <item row="0" column="2">
                <widget class="QPushButton" name="btn_rem_x64_junk">
                 <property name="text">
                  <string>Asm X64 junk</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="btn_debian_base">
                 <property name="text">
                  <string>Debian Base</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btn_debian_coreutils">
                 <property name="toolTip">
                  <string>Remove strings from debian coreutils binaries</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>2</number>
                 </property>
                 <property name="text">
                  <string>Debian coreutils</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="btn_rem_x86_junk">
                 <property name="text">
                  <string>Asm x86 junk</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="btn_rem_go_str">
                 <property name="text">
                  <string>Go imports</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_encoder">
           <attribute name="title">
            <string>Encoder</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QGridLayout" name="gridLayout_21">
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>reset</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="btn_save_to_binary">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save binary file as</string>
                 </property>
                 <property name="icon">
                  <iconset theme="media-floppy"/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btn_save_xored">
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save XORed data as</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>ASCII</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_13">
               <item row="0" column="0">
                <widget class="QPlainTextEdit" name="txt_encode_ascii"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>HEX</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_15">
               <item row="0" column="0">
                <widget class="QPlainTextEdit" name="txt_encode_hex"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Base SitxyFour</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_14">
               <item row="0" column="0">
                <widget class="QPlainTextEdit" name="txt_encode_base64"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Binary</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_16">
               <item row="0" column="0">
                <widget class="QPlainTextEdit" name="txt_encode_bin"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>X0R (Result ASCII only)</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_17">
               <item row="0" column="0">
                <widget class="QPlainTextEdit" name="txt_encode_xor"/>
               </item>
               <item row="0" column="1">
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>KEY</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_12">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Decimal</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="txt_spinbox_xor_key_dec">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>HEX</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="txt_encode_xorkey_hex"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="actionTo_JSON"/>
     <addaction name="actionTo_txt"/>
    </widget>
    <addaction name="actionOpen_new_file"/>
    <addaction name="menuExport"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionQt_Style_clear"/>
     <addaction name="actionQt_Style_dark"/>
     <addaction name="actionQt_Pink"/>
     <addaction name="action_by_ChatGPT"/>
     <addaction name="actionBlack_White"/>
    </widget>
    <widget class="QMenu" name="menuFont_size">
     <property name="title">
      <string>Font size</string>
     </property>
     <addaction name="font_zoom"/>
     <addaction name="font_dezoom"/>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="menuFont_size"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen_new_file">
   <property name="text">
    <string>Open new file</string>
   </property>
  </action>
  <action name="actionTo_JSON">
   <property name="text">
    <string>To JSON</string>
   </property>
  </action>
  <action name="actionTo_txt">
   <property name="text">
    <string>To txt</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionQt_Style_clear">
   <property name="text">
    <string>Qt Style clear</string>
   </property>
  </action>
  <action name="actionQt_Style_dark">
   <property name="text">
    <string>Qt Style dark</string>
   </property>
  </action>
  <action name="actionQt_Pink">
   <property name="text">
    <string>Qt Pink</string>
   </property>
  </action>
  <action name="action_by_ChatGPT">
   <property name="text">
    <string> Random by ChatGPT </string>
   </property>
  </action>
  <action name="actionBlack_White">
   <property name="text">
    <string>Black &amp; White</string>
   </property>
  </action>
  <action name="font_zoom">
   <property name="icon">
    <iconset theme="zoom-in"/>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="font_dezoom">
   <property name="icon">
    <iconset theme="zoom-out"/>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
